man/fi_endpoint.3.md

```diff
@@ -264,9 +264,11 @@ return -FI_EBUSY.
 
 Outstanding operations posted to the endpoint when fi_close is
 called will be discarded.  Discarded operations will silently be dropped,
-with no completions reported.  Additionally, a provider may discard previously
-completed operations from the associated completion queue(s).  The
-behavior to discard completed operations is provider specific.
+with no completions reported. Memory buffers associated with the endpoint via
+transmission interfaces must be kept alive until a completion is
+generated or fi_close(ep) returns.  Additionally, a provider may discard
```

Comment:
Who is responsible for keeping the buffers alive, the application or the provider? What exactly is "keeping the buffer alive" means: buffer is not freed, MR is not deregistered, and/or anything else?

Reply:
Who:
The user/library that supplied Libfabric with buffers via data transfer methods (i.e.. whoever called send/recv/write/read)

What:

Bufferes cannot be freed, deregistered or re-used until after fi_close(ep) returns.

I can update to make this more clear.  Are you in favor of this PR with the clarification?

Reply:
Yes, clarification is preferred.

Reply:
ok, updated.